### PR TITLE
React-codemod to refactor React imports

### PR DESCRIPTION
**Pull request by sytranvn**
https://github.com/reactjs/react-codemod
As of Babel 7.9.0, when using runtime: automatic in @babel/preset-react or @babel/plugin-transform-react-jsx, you will not need to explicitly import React for compiling jsx. (next supports this out of the box)
- This PR ran codemod to remove the redundant import statements. 
You can try it yourself against master by running `npx react-codemod update-react-imports`

- Also fix mixing of single and double quotes among imports. 

After refactor the page is still working. 

image
image

### CODE REVIEW

**Comment by sytranvn** [comment]
@gabek can you review this PR?

**Comment by gabek** [comment]
Hey @sytranvn!  While this does clean up some specific files, it would actually make sense to put changes in place where a linter and/or formatter standardizes these rules going forward. 

Would you be able to use what you're using for this single patch to fix these issues going forward?

**Comment by sytranvn** [comment]
@gabek I added new rules to eslintrc to use single quotes in JS/TS and double quotes in JSX.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build && next export",
-    "start": "next start"
+    "start": "next start",
+    "lint": "tsc --noEmit && eslint -c .eslintrc.js './**/*.{js,ts,tsx}'"
```
Comment (sytranvn) [inline package.json:9]: There are many bypassed Typescript errors. Adding lint check on PRs would certainly not a bad idea. But we need to fix these first.

image

Comment (sytranvn) [reply package.json:9]: I don't use next. So not one hundred percent sure why it's ignoring TS errors when we run build.

**Comment by sytranvn** [comment]
Sorry I din't have time to continue the work.

**Pull request closed without merging**